lighthouse-core/lib/emulation.js

```diff
@@ -7,14 +7,6 @@
 
 /** @typedef {import('../gather/driver.js')} Driver */
 
-const OFFLINE_METRICS = {
-  offline: true,
-  // values of 0 remove any active throttling. crbug.com/456324#c9
```

Comment:
this seems like the only tricky knowledge that could be lost if trying to recreate going offline, but based on that comment is it still true/useful today?

Reply:
Based on the [immediate followup commit](https://codereview.chromium.org/1039213002/patch/1/10001), it looks like future attempts should be fine with just `offline: true`.

If we ever need to find this again, we know the file to use git history on :)